### ISSUE TITLE
Error: Module es6-promise-plugin.Promise does not exist issue 

### ISSUE THREAD

**Issue by natalinacinovicmrf** [vstirbu/PromisesPlugin#9]
I made an update to the SocialSharing-PhoneGap-Plugin and it now has a new dependency on this plugin (es6-promise-plugin).

The issue I am experiencing is that when I try to load the app I receive an error:

`Error: Module es6-promise-plugin.Promise does not exist.`
in this function of the native file

```
function addEntry (strategy, moduleName, symbolPath, opt_deprecationMessage) {
    if (!(moduleName in moduleMap)) {
        throw new Error('Module ' + moduleName + ' does not exist.');
    }
    symbolList.push(strategy, moduleName, symbolPath);
    if (opt_deprecationMessage) {
        deprecationMap[symbolPath] = opt_deprecationMessage;
    }
}
```

Do you know if I need to do anything else, note the plugin somewhere else?
Thanks!

**Comment by vstirbu** [comment]
have you updated it by uninstalling and installing your plugin again?

**Comment by natalinacinovicmrf** [comment]
Yes, before I had an older version of that  plugin (like from 2 years ago) which of course didn't have that dependency. So now it was added for the first time.

**Comment by vstirbu** [comment]
This plugin is pretty simple as it just patches the global object (window) with the Promise if not already available. It is used [elsewhere](https://github.com/apache/cordova-plugin-screen-orientation/blob/13b34f05e6abf454f2cb7f1b7870c0cb31cc6a32/plugin.xml#L55) and installed by pretty many users.

I've noticed that you opened an issue with SocialSharing-PhoneGap-Plugin. You might try to create a new project and try to replicate the issue there. Make sure the problem is not caused by your environment.
